NOTICE.txt

```diff
@@ -0,0 +1,30 @@
+==============================================================
+ Apache Solr
+ Copyright 2006-2021 The Apache Software Foundation
+==============================================================
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+Includes software from other Apache Software Foundation projects,
+including, but not limited to:
+  - Apache Lucene Java
```

Comment:
So this project doesn't actually contain software from Apache Lucene, right? We use the APIs, but not the libraries.

Reply:
I imagine we are going to have to fill this out with the go dependencies we use?

Reply:
Yes, but we might not have to add Solr as it's the same project. I think the Lucene bit that exists in 'Solr' is just cruft from 10 years ago :) It'll be required again in the future, but for the operator, we shouldn't need Lucene in this list.